.gitmodules

```diff
@@ -22,6 +22,31 @@
 [submodule "data/hssd-hab"]
 	path = data/hssd-hab
 	url = https://huggingface.co/datasets/hssd/hssd-hab
+[submodule "src/home_robot/home_robot/perception/detection/minigpt4/MiniGPT-4"]
```

Comment:
@keke-220 Can we remove GPT stuff from this release?

Reply:
yes we can